LM-774 default env vars for AppDynamics (#528)
mendix/cf-mendix-buildpack: README.md

Please note that AppDynamics requires Mendix 6.2 or higher.

| Environment Variable | Default | Description |
|-|-|-|
| `APPDYNAMICS_CONTROLLER_PORT` | 443 | Port of AppDynamics controller |
| `APPDYNAMICS_CONTROLLER_SSL_ENABLED` | true | Set if SSL is required |
| `APPDYNAMICS_CONTROLLER_HOST_NAME` | - | Name of AppDynamics host without 'http://' |
| `APPDYNAMICS_AGENT_APPLICATION_NAME` | App name as on Dev. Portal | Name of the app to be displayed on Controller UI |
| `APPDYNAMICS_AGENT_ACCOUNT_NAME` | - | See 'License/Account' on the Controller UI |
| `APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY` | - | See 'License/Account' on the Controller UI |

> | `APPDYNAMICS_AGENT_NODE_NAME` * | node | How a node is displayed on the Controller UI |

Reviewer: Would `node` be the hostname ? Maybe an example would help.

Author: Added a description and example column

| `APPDYNAMICS_AGENT_TIER_NAME` | App Environment UUID | How a tier is displayed on the Controller UI |

\* The `APPDYNAMICS_AGENT_NODE_NAME` environment variable will be appended with the value of the `CF_INSTANCE_ID` variable. If you use `mynode` for `APPDYNAMICS_AGENT_NODE_NAME` , the AppDynamics agent will be configured as `mynode-0` for instance `0` and `mynode-1` for instance `1` , etc.

For more details about nodes and tiers: [Tiers and Nodes](https://docs.appdynamics.com/22.1/en/application-monitoring/tiers-and-nodes).

Required variables: `APPDYNAMICS_AGENT_ACCOUNT_NAME`, `APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY` and `APPDYNAMICS_CONTROLLER_HOST_NAME`.

If you have all the required environment variables specified above, the AppDynamics Java Agent will be configured for your application.

> The other variables (not required) will be set to the default values.

Reviewer: ```suggestion
The other environment variables are optional and will be set to the default values.
```

Author: done

> Please note that AppDynamics requires Mendix 6.2 or higher.

Reviewer 2: The buildpack only supports Mendix >= 7.

Author: Changed to 7.15. As in main documentation
